core/src/main/resources/hudson/triggers/Messages_de.properties

```diff
@@ -25,5 +25,8 @@ SCMTrigger.getDisplayName={0} Abfrage-Protokoll
 SCMTrigger.BuildAction.DisplayName=Abfrage-Protokoll
 SCMTrigger.SCMTriggerCause.ShortDescription=Build wurde durch eine SCM-�nderung ausgel�st.
 TimerTrigger.DisplayName=Builds zeitgesteuert starten
+TimerTrigger.MissingWhitespace=Es scheinen Leerzeichen zwischen * und * zu fehlen.
+TimerTrigger.no_schedules_so_will_never_run=Keine zeitgesteuerten Builds, somit wird Job nie laufen
```

Comment:
Maybe this translation is better:
TimerTrigger.no_schedules_so_will_never_run=Es sind keine zeitgesteuerten Builds vorhanden, somit wird der Job nie ausgef\u00fchrt werden.